support/Procfile

```diff
@@ -1,3 +1,4 @@
+web: support/builder_web.sh
```

Comment:
This should probably be changed to `hab svc start core/builder-api-proxy`

Reply:
What is `builder-api-proxy` actually needed for? It seems to works just fine w/o (and is not mentioned in the new, authoritative [dev guide](https://github.com/habitat-sh/habitat/blob/master/BUILDER_DEV.md) nor in the [builder-web README](https://github.com/habitat-sh/habitat/blob/master/components/builder-web/README.md)).